tests/unit/jobserver/test_middleware.py

```diff
@@ -77,6 +77,11 @@ def test_client_ip_middleware_proxied(rf):
     middleware(request)
     assert request.backend == backend
 
+    # tpp client via proxy
```

Comment:
This wouldn't normally happen without spoofing - the immeadiate proxy address wouldn't be in X-Forwarded-For.

So I think this case is covered in the other test.

Reply:
yep understand now